PhDThesisPSnPDF.cls

```diff
@@ -1114,14 +1124,18 @@ wish to left align your text}
     \renewcommand{\printthesisindex}{}
     \renewcommand{\bibname}{}
     \renewcommand{\bibliography}[1]{\thispagestyle{empty}}
+    
+    % Turn on name and title on abstract (DYC 2020)
+    \ntabstracttrue
 \else
     \defineAbstractfalse
 \fi
 
-
 \newenvironment{abstract} {
-\ifPHD@abstract
-% Separate abstract as per Student Registry guidelines
+\ifntabstract
```

Comment:
Since this is set as default, we don't need it here

Reply:
This is still needed for the "compile the entire thesis and don't put my name and thesis title onto the abstract page" combination to work.